client/components/editor/TeachingElement.vue

```diff
@@ -24,7 +39,48 @@ export default {
     disabled: { type: Boolean, default: false },
     dragged: { type: Boolean, default: false }
   },
+  data() {
+    return {
+      isFocused: false
+    };
+  },
+  computed: {
+    ...mapGetters('activeUsers', ['activeUsers']),
+    ...mapState('activeUsers', ['sseId']),
+    contentActiveUsers() {
```

Comment:
Consider renaming to something like elementActivity or something along those lines. I understand that we have concept of activity which clashes with this feature.